### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by DarkVlad16**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |  Fixed old version of inputmask

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
         "ezyang/htmlpurifier": "^4.6",
         "cebe/markdown": "~1.0.0 | ~1.1.0 | ~1.2.0",
         "bower-asset/jquery": "3.6.*@stable | 3.5.*@stable | 3.4.*@stable | 3.3.*@stable | 3.2.*@stable | 3.1.*@stable | 2.2.*@stable | 2.1.*@stable | 1.11.*@stable | 1.12.*@stable",
-        "bower-asset/inputmask": "~3.2.2 | ~3.3.5",
+        "bower-asset/inputmask": "*",
```
Comment (samdark) [inline composer.json:79]: [Using `*` is dangerous](https://thephp.cc/articles/the-death-star-version-constraint). It's better to add another version.

Comment (DarkVlad16) [reply composer.json:79]: > [Using `*` is dangerous](https://thephp.cc/articles/the-death-star-version-constraint). It's better to add another version.

Okay, but why unit tests failed on php8?

Comment (schmunk42) [reply composer.json]: > Okay, but why unit tests failed on php8?

Failures are not InputMask related. 
https://github.com/yiisoft/yii2/actions/runs/4041727523/jobs/6948596350#step:11:149

**Comment by schmunk42** [comment]
> So, May you add this pull request to master version? Cause old version of inputmask not supports multiple masks and other features. Or how can I override it, to use new version?

To override it with a version which is actually not compatible you can use 

```
  "bower-asset/inputmask": "5.0.1 as 3.3.999"
```

But it's a hack, you need to make sure it is compatible with the widget.

**Comment by terabytesoftw** [comment]
Closed in favor https://github.com/yiisoft/yii2/pull/19963

**Pull request closed without merging**